src/track.rs

```diff
@@ -288,7 +288,7 @@ where
 
 /// Destructure a `Tracked<_>` into its parts.
 #[inline]
-pub fn to_parts_ref<T>(tracked: Tracked<T>) -> (&T, Option<&T::Constraint>)
+pub fn to_parts_ref<'a, T>(tracked: &Tracked<'a, T>) -> (&'a T, Option<&'a T::Constraint>)
```

Comment:
I think this change is unnecessary since `Tracked` remains `Copy`.

Reply:
Yes, it was needed when I tested making it `Clone`